tdrs-backend/tdpservice/settings/common.py

```diff
@@ -270,9 +270,8 @@ class Common(Configuration):
     # Sessions
     SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
     SESSION_COOKIE_HTTPONLY = True
-    SESSION_TIMEOUT = 30
     SESSION_EXPIRE_AT_BROWSER_CLOSE = True
-    SESSION_COOKIE_AGE = 30 * 60  # 30 minutes
+    SESSION_COOKIE_AGE = 15 * 60  # 15 minutes
```

Comment:
find out what prod exp is

Reply:
Nit: If @ADPennington want to have extended session cookie, then you should be able to set the session at login to a different value for admin users.